.eslintrc.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
     ],
     '@typescript-eslint/no-non-null-assertion': 1,
     // Disable error on devDependencies importing since this isn't a TS library
+    'no-param-reassign': ['error', { props: false }],
```

Comment:
AFAICT, this can cause unwanted issues, and we should probably consider keeping the rule on to reduce the chance of mistakes. Unless there's a reason why not to.